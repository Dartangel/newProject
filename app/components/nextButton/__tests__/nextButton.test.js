import 'react-native';
import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import NextButton from '../index';

describe('NextButton', () => {
    test('NextButton snapshot', () => {
        const renderer = new ShallowRenderer();
        const snapshot = renderer.render(<NextButton />);
        expect(snapshot).toMatchSnapshot();
    });
});