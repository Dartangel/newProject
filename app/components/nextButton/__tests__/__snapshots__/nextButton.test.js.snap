// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NextButton NextButton snapshot 1`] = `
<View>
  <TouchableOpacity
    activeOpacity={0.2}
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#08DBA5",
        "borderRadius": 50,
        "justifyContent": "center",
        "marginVertical": 20,
        "minHeight": 60,
        "paddingHorizontal": 15,
      }
    }
  >
    <Text
      style={
        Object {
          "color": "black",
          "fontSize": 20,
        }
      }
    />
  </TouchableOpacity>
</View>
`;
